cmd/govim/test_config.go

```diff
@@ -75,6 +79,13 @@ func (v *vimstate) dumpPopups(args ...json.RawMessage) (interface{}, error) {
 	return sb.String(), nil
 }
 
+func (v *vimstate) showMessagePopup(args ...json.RawMessage) (interface{}, error) {
+	ctx := context.Background()
```

Comment:
Yep, this is exactly the kind of function we will need.

But what we need to do is asynchronously trigger the message in a go routine. 

You can return whatever you like, because the popup will be shown asynchronously.